app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -82,14 +82,22 @@ function initializePodcastPlayback() {
 
   function spinPodcastRecord(customMessage) {
     if (audioExistAndIsPlaying() && recordExist()) {
-      getById(`record-${window.activeEpisode}`).classList.add('playing');
+      var podcastPlaybackButton = getById(`record-${window.activeEpisode}`);
+      podcastPlaybackButton.classList.add('playing');
+      var isPressed =
+        podcastPlaybackButton.getAttribute('aria-pressed') === 'true';
+      podcastPlaybackButton.setAttribute('aria-pressed', !isPressed);
```

Comment:
```suggestion
      podcastPlaybackButton.setAttribute('aria-pressed', 'true');
```

I think in this case the podcast is _always_ playing, right? In which case I think let's just explicitly make sure this is set to 'true', then there's no chance of it being out of step / remove the need to check the current attribute value.